convert/fix.go

```diff
@@ -128,15 +128,27 @@ func BookActionToFIX(action bitfinex.BookAction) enum.MDUpdateAction {
 	return enum.MDUpdateAction_NEW
 }
 
+// MTSToTime converts bitfinex millisecond timestamp to go time.Time
+func MTSToTime(mts int64) (time.Time, bool) {
+	if mts > 0 {
+		return time.Unix(0, mts*1000000), true
+	}
+	return time.Time{}, false
+}
+
 // TimeInForceToFIX converts bitfinex order type to FIX TimeInForce
-func TimeInForceToFIX(ordtype bitfinex.OrderType) enum.TimeInForce {
+func TimeInForceToFIX(ordtype bitfinex.OrderType, mtstif int64) (enum.TimeInForce, time.Time) {
+	tif, ok := MTSToTime(mtstif)
+	if ok {
+		return enum.TimeInForce_GOOD_TILL_DATE, tif
+	}
 	switch ordtype {
 	case bitfinex.OrderTypeFOK:
 		fallthrough
 	case bitfinex.OrderTypeExchangeFOK:
-		return enum.TimeInForce_FILL_OR_KILL
```

Comment:
Please could you add field mappings to the README

Reply:
Done [https://github.com/bitfinexcom/bfxfixgw/pull/19/commits/2a96c9f88e8a63e54801f5c4745201b5f837f117](https://github.com/bitfinexcom/bfxfixgw/pull/19/commits/2a96c9f88e8a63e54801f5c4745201b5f837f117)